File: frontend/replace.js
const fs = require('fs');
const path = require('path');
const glob = require('glob'); // Not using glob package to keep it simple, I'll just use the list

const filesToUpdate = [
    "src/app/history/page.tsx",
    "src/app/sync/page.tsx",
    "src/app/sources/page.tsx",
    "src/app/scan-history/[scanId]/page.tsx",
    "src/app/dashboard/page.tsx",
    "src/app/companies/page.tsx",
    "src/app/api-config/page.tsx",
    "src/features/communicationTPR/components/Layout.tsx"
];

for (const file of filesToUpdate) {
    const fullPath = path.join('c:/Users/aksv4/Desktop/Job-Finder/frontend', file);
    if (fs.existsSync(fullPath)) {
        let content = fs.readFileSync(fullPath, 'utf8');
        content = content.replace(/max-w-7xl mx-auto/g, 'w-full max-w-none');
        content = content.replace(/mx-auto max-w-7xl/g, 'w-full max-w-none');
        fs.writeFileSync(fullPath, content, 'utf8');
        console.log(`Updated ${file}`);
    } else {
        console.log(`Not found: ${file}`);
    }
}

File: frontend/replace.py
import os
import glob

# Files to update
files_to_update = [
    r"src\app\history\page.tsx",
    r"src\app\sync\page.tsx",
    r"src\app\sources\page.tsx",
    r"src\app\scan-history\[scanId]\page.tsx",
    r"src\app\dashboard\page.tsx",
    r"src\app\companies\page.tsx",
    r"src\app\api-config\page.tsx",
    r"src\features\communicationTPR\components\Layout.tsx"
]

for file_path in files_to_update:
    full_path = os.path.join(r"c:\Users\aksv4\Desktop\Job-Finder\frontend", file_path)
    if os.path.exists(full_path):
        with open(full_path, 'r', encoding='utf-8') as f:
            content = f.read()
        
        # Replace max-w-7xl mx-auto with w-full
        new_content = content.replace('max-w-7xl mx-auto', 'w-full max-w-none')
        # Handle cases where it might be written slightly differently
        new_content = new_content.replace('mx-auto max-w-7xl', 'w-full max-w-none')
        
        with open(full_path, 'w', encoding='utf-8') as f:
            f.write(new_content)
        print(f"Updated {file_path}")
    else:
        print(f"Not found: {file_path}")

# Let's also do communicationTPR pages if any
comm_pages = glob.glob(r"c:\Users\aksv4\Desktop\Job-Finder\frontend\src\app\communication-tpr\**\page.tsx", recursive=True)
for full_path in comm_pages:
    with open(full_path, 'r', encoding='utf-8') as f:
        content = f.read()
    new_content = content.replace('max-w-7xl mx-auto', 'w-full max-w-none')
    new_content = new_content.replace('mx-auto max-w-7xl', 'w-full max-w-none')
    with open(full_path, 'w', encoding='utf-8') as f:
        f.write(new_content)
    print(f"Updated comm: {full_path}")
